test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -778,10 +779,18 @@ protected Settings featureFlagSettings() {
         for (Setting builtInFlag : FeatureFlagSettings.BUILT_IN_FEATURE_FLAGS) {
             featureSettings.put(builtInFlag.getKey(), builtInFlag.getDefaultRaw(Settings.EMPTY));
         }
-        featureSettings.put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
         return featureSettings.build();
     }
 
+    /**
+     * Setting Telemetry feature flag settings at base IT. We don't need this to be overridden by
+     * individual test cases.
+     * @return Telemetry Feature flag settings.
+     */
+    private Settings telemetryFeatureFlagSettings() {
```

Comment:
This is not needed, we have `featureFlagSettings()` that could be overridden, please remove

Reply:
Same reason, we want these settings to be enabled always. Cannot use featureFlagSettings() which can be overridden in individual test cases

Reply:
The override mechanism allows to isolate tests from the components they don't need, we have to respect that. If some tests don't need default feature flags (and telemetry) - we should provide the way to do so.

Reply:
Ack. Removed this telemetryFeatureFlagSetting